lib/streamlit/config.py

```diff
@@ -202,6 +202,19 @@ def _delete_option(key):
 )
 
 
+_create_option(
+    "global.disableWatchdog",
+    description="""
+        By default, Streamlit detects file changes with the Python watchdog
+        module. Optionally, you can force Streamlit to use Polling even if 
+        watchdog is available. This is not recommended.
+
+        If you'd like to use polling, set this to True.
+    """,
+    default_val=False,
+    type_=bool,
+)
```

Comment:
This should be in the "server" section, since that's where all other watchdog-related configs live today.

Reply:
Yeah I makes sense, I just saw the `global.disableWatchdogWarning` and put it alongside that.

Reply:
Oh, I see...

Turns out `global` is a weird place for `disableWatchdogWarning` too :D

I would move it somewhere else at some point, but that takes a little more work since we'd have to deal with the deprecation. So just leave `disableWatchdogWarning` as is for now and put the above `fileWatcherType` inside `server`, please.

Naming is hard :(